src/main/java/frc/robot/constants/IDs.java

```diff
@@ -11,4 +13,10 @@ public class IDs {
 		PowerDistribution.ModuleType.kRev
 	);
 
+	// ! need to check those IDs which doesn't exists yet since we don't have a working robot
+
+	public static final SparkMaxDeviceID ELEVATOR_FIRST_MOTOR = new SparkMaxDeviceID(2, CANSparkLowLevel.MotorType.kBrushless);
```

Comment:
Find better indicator than first and sec (pose on robot can be a good one)

Reply:
they're next to each other, no one is higher/lower than the other, I thought about that but I can't think about an name.

Reply:
left right?

Reply:
left/right in proportion to what? I can stand from a different side